system/platform-core/src/main/java/org/platformlambda/core/system/WorkerHandler.java

```diff
@@ -133,7 +133,7 @@ public void executeFunction(EventEnvelope event) {
                     payload.put(TRACE, metrics);
                     po.send(dt.setBody(payload));
                 }
-            } catch (Exception e) {
```

Comment:
the additional error types are not required because it is solely for distributed tracing.

Please check "Exception" to "IllegalArgumentException" that is the only exception from po.send